tests/test_bidsInterface.py

```diff
@@ -64,9 +68,20 @@ def readLocalDicomIncremental(volIdx, entities):
 def dicomStreamTest(bidsInterface):
     # initialize the stream
     entities = {'subject': '01', 'task': 'test', 'run': 1, 'suffix': 'bold', 'datatype': 'func'}
+
+    # Try to init a stream to a non-allowed path
+    if bidsInterface.isRunningRemote():
+        with pytest.raises(Exception) as err:
```

Comment:
The test case might be more clear if the specific exception type that's expected is in the `pytest.raises` block. It might also help debug, as then the `pytest.raises` will fail instead of the `assert` when the wrong exception is thrown.

Reply:
I originally tried that, but in the rpc case the exception is a derived rpc object, so it seemed simpler to just catch all exceptions.